oca/recipe/odoo/base.py

```diff
@@ -537,12 +537,15 @@ def read_requirements_pip_after_v8(self, req_path, versions, develops):
         else:
             from pip._internal.req import parse_requirements
         for inst_req in parse_requirements(req_path, session=fake_session):
-            if pip_version() < (20, 0, 0):
+            if hasattr(inst_req, 'req'): # pre 20.1b1
```

Comment:
need this for the outlier 20 version ubuntu uses: https://packages.ubuntu.com/focal/python3-pip